dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/LibertyLoader.java

```diff
@@ -83,16 +84,22 @@ protected final Class<?> findClass(String className) throws ClassNotFoundExcepti
     protected abstract Class<?> loadClass(String className, boolean resolve, DelegatePolicy delegatePolicy, boolean returnNull) throws ClassNotFoundException;
 
     protected abstract Class<?> findClass(String className, DelegatePolicy delegatePolicy, boolean returnNull) throws ClassNotFoundException;
-    
-    @Override
-    protected URL findResource(String resName) {
-        return super.findResource(resName);
```

Comment:
I don't understand why you removed it.  Could just update it to return null.  The point of it being here is to put it in the stack trace.